main_test.go

```diff
@@ -16,34 +16,32 @@ func TestFindRoot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	expect := filepath.Join(wd, "_testdata", "rootfind")
-
-	got1, err := findProjectRoot(expect)
+	want := filepath.Join(wd, "_testdata", "rootfind")
+	got1, err := findProjectRoot(want)
 	if err != nil {
 		t.Errorf("Unexpected error while finding root: %s", err)
-	} else if expect != got1 {
-		t.Errorf("findProjectRoot directly on root dir should have found %s, got %s", expect, got1)
+	} else if want != got1 {
```

Comment:
want is canonical.

I'd not only rely on the code review doc (given it is very limited to explain the go project's style itself) but ask a core project reviewer (via golang-dev) to give readability reviews.

Some repos merged to the Go project such as godoc.org have become a problem over time due to its distinct style. The changes are low that the other Go contributors like to contribute to the repos with distinct style, godoc.org is suffering more from lack of maintenance even though it is a high impact project.